packages/firestore/test/unit/local/globals_cache.test.ts

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Is there a schema upgrade test for web? I think so..can you check?

Reply:
Added missing migration tests: 14>15, 15>16 and 16>17.